### DISCUSSION TITLE
how do I make a nested url without nested layout?

### DISCUSSION THREAD

**Discussion by k11q** [kiliman/remix-flat-routes#66] [Q&A] [answered]
e.g. route/products/$productId , I dont want the productId page to inherit the layout from products

**Comment by kiliman** [comment, marked as answer]
You can name it `_.$productId`

This means _append the `_` to the parent_, so it's like you did `products_.$productId`

**Reply by k11q** [reply]
Thanks! So intuitive!

**Comment by rodolphoasb** [comment]
@kiliman, sorry to flag you but I tried a lot of things and no dice.

I'm using `remix-flat-routes`. How can I opt-out from the _db layout on the emails/new route?

![image](https://github.com/kiliman/remix-flat-routes/assets/8127858/1dbaa70e-c2d9-4b0c-8f71-03b7de5a93be)

I want to have the dashboard layout on _index but not on emails/new.

**Reply by kiliman** [reply]
Unfortunately, the only way is to create a top-level `emails.new` route.

By explicitly nesting under `_db`, you're opting into that layout. 

BTW: it looks like you're using *hybrid* routes. Although technically, `remix-flat-routes` lets you use naked folders, you should only use that for the colocated folder. The internal folders should have the `+` suffix. That is, your routes should look like this:
```
app/routes
├── _db+
│   ├── _layout.tsx
│   ├── customers+
│   │   └── _index
│   │       └── route.tsx
│   └── emails+
│       └── _index
│           └── route.tsx
└── emails.new
    └── route.tsx
```
I will likely deprecate that in a future version as it confuses nesting vs colocation.

**Reply by rodolphoasb** [reply]
Gotcha, really appreciate your help and explanation! Remix FTW!
